src/helpers/iconpicker.class.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+var $           = require('jquery');
+var eventablejs = require('eventablejs');
+var _           = require('../lodash.js');
+var Modal       = require('etudiant-mod-modal');
+var xhr         = require('etudiant-mod-xhr');
+var Slider      = require('./slider.class.js');
+var FileUploader = require('../extensions/file-uploader.js');
+
+var iconPickerHtml = '<div class="icon-picker"><div class="droppable st-block__upload-container">' + i18n.t('blocks:illustrated:placeholder:drop') + '</div></div>';
+
+function triggerChangeIllustratedPicture(block, pictureInformations){
+    block.blockRef.imageId = pictureInformations.id;
+    block.trigger('picture:change', pictureInformations);
+
+    block.modal.close();
+
+}
+
+function bindClickOnIcons(block) {
+    $.each(block.slider.$elem.find('img'), function(){
+        $(this).on('click', function(){
+            triggerChangeIllustratedPicture(block, {
+                src: $(this).attr('src'),
+                copyright: $(this).attr('alt'),
+                id: $(this).data('id')
+            });
+        });
+    });
+}
+
+function dropEvent(iconPicker){
+    $(iconPicker.modal.$elem.children('.droppable')[0]).off();
```

Comment:
@arthur-balagne unless I am very much mistaken, this line of code does strictly nothing no?